pkg/util/pods.go

```diff
@@ -92,6 +92,22 @@ func MergeVolumeMounts(original, additional []v1.VolumeMount) []v1.VolumeMount {
 	return original
 }
 
+func MergeVolumeDevices(original, additional []v1.VolumeDevice) []v1.VolumeDevice {
```

Comment:
shall we check against the case that devicePath collides with normal volumeMounts path?

Reply:
No, our generic method validateContainersForSidecarSet is already validated.